scripts/microbe_stage/microbe.as

```diff
@@ -1038,7 +1021,7 @@ class MicrobeSystem : ScriptSystem{
             {
                 auto playerSpecies = MicrobeOperations::getSpecies(world, "Default");
                 if (!microbeComponent.isPlayerMicrobe &&
-                    microbeComponent.speciesName != playerSpecies.name)
+                    microbeComponent.species.name != playerSpecies.name)
```

Comment:
Maybe this could be converted to `microbeComponent.species is playerSpecies` to just compare the handles instead of do a string comparison.